src/mad_spark_alt/core/smart_orchestrator.py

```
@@ -169,14 +171,17 @@ async def run_qadi_cycle(
+        await self._synthesize_conclusion(result, problem_statement, context)
+
         end_time = time.time()
         result.execution_time = end_time - start_time
 
         logger.info(
             f"Smart QADI cycle {cycle_id} completed in {result.execution_time:.2f}s"
         )
-        if total_llm_cost > 0:
-            logger.info(f"Total LLM cost: ${total_llm_cost:.4f}")
+        if result.llm_cost > 0:
+            logger.info(f"Total LLM cost: ${result.llm_cost:.4f}")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)